config/environments/development.rb

```diff
@@ -31,7 +31,20 @@
   config.active_storage.service = :local
 
   # Don't care if the mailer can't send.
-  config.action_mailer.raise_delivery_errors = false
+  config.action_mailer.perform_caching = false
+
+  config.action_mailer.raise_delivery_errors = true
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.default_url_options = {host: "localhost:3000"}
+# SMTP settings for gmail
+  config.action_mailer.smtp_settings = {
+    :address => "smtp.gmail.com",
```

Comment:
dùng cú pháp sau, fix cho các line còn lại
```suggestion
    address: "smtp.gmail.com",
```